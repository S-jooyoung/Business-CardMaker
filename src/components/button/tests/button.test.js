// import dependencies
import React from "react";

// import custom-component
import Button from "../button";

// import snapshot-testing methods
import renderer from "react-test-renderer";

describe("Button", () => {
  let name = "테스트";
  let onClick;
  let buttonComponent;

  beforeEach(() => {
    onClick = jest.fn();
    buttonComponent = <Button name={name} onClick={onClick} />;
  });

  it("render", () => {
    const component = renderer.create(buttonComponent).toJSON();
    expect(component).toMatchSnapshot();
  });
});
