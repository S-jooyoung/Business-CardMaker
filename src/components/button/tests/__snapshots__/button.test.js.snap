// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `
<button
  className="button"
  onClick={[MockFunction]}
>
  테스트
</button>
`;
